client/src/Components/PrivateRoutes/AuthenticatedRoutes.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from "react";
 import { Switch, withRouter, Route, Redirect } from "react-router-dom";
 import { connect } from "react-redux";
+import axios from 'axios';
```

Comment:
so im not sure about this one, personally i thing it's kinda weird making an axios call in a route but i had also thought that making a route a stateful component was weird. would it be possible to set the user info on state from the login? idk best practice, maybe we see what Donovan has to say on that.